inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -2071,7 +2082,7 @@ GNAPluginNS::ConnectionDetails GNAGraphCompiler::connectInput(CNNLayerPtr layer,
                 gnamem->reserve_ptr(&memoryLayer.gna_ptr, ALIGN64(memorySize), 64);
                 gnamem->bind_ptr(ptr, &memoryLayer.gna_ptr, offset);
             } else {
-                if (num_data_bytes_in > memorySize) {
+                if (num_data_bytes_in > memorySize - offset) {
```

Comment:
this looks more restrictive requirement, so i expect some test to hit that, because old tests cannot - am i correct and we have test for this?

Reply:
My LSTM test will fail without that

Reply:
Separate task was created to make tests. It will be separate PR